api/app/Policies/UserPolicy.php

```diff
@@ -205,6 +205,16 @@ public function restore(User $user, User $model)
         return $user->isAbleTo('delete-any-user') && $user->id !== $model->id; // Do not allow user to restore their own model.
     }
 
+    /**
+     * Determine whether the user can wipe work email information.
+     *
+     * @return \Illuminate\Auth\Access\Response|bool
+     */
+    public function wipeWorkEmailInformation(User $user, User $model)
+    {
+        return $user->isAbleTo('update-own-user') && $user->id === $model->id;
+    }
```

Comment:
Mutation can easily be used by other users like say platform admins in the future, just need to update this

Reply:
Yeah I think it should be distinct, it is a very specific and special operation